File: web/src/main/java/uk/co/rafearnold/randomworkoutroutine/web/controller/GroupController.java
package uk.co.rafearnold.randomworkoutroutine.web.controller;

import uk.co.rafearnold.randomworkoutroutine.web.model.entity.Group;
import uk.co.rafearnold.randomworkoutroutine.web.service.GroupService;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/api/group")
public class GroupController extends ItemController<Group> {

    public GroupController(GroupService service) {
        super(service);
    }
}

File: web/src/main/java/uk/co/rafearnold/randomworkoutroutine/web/controller/GroupController.kt
package uk.co.rafearnold.randomworkoutroutine.web.controller

import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController
import uk.co.rafearnold.randomworkoutroutine.web.model.entity.Group
import uk.co.rafearnold.randomworkoutroutine.web.service.GroupService

@RestController
@RequestMapping("/api/group")
class GroupController(service: GroupService) : ItemController<Group>(service)
